### DISCUSSION TITLE
How to change sources now that they are cached?

### DISCUSSION THREAD

**Discussion by Galunid** [lncrawl/lightnovel-crawler#1081] [Q&A] [answered]
Basically I changed stuff in `<project_dir>/sources/*`, but running `python <project_dir>/lncrawl` was using sources cached in `~/.lncrawl/sources`, is there some new option to specify sources dir?

**Comment by dipu-bd** [comment, marked as answer]
have you pulled the latest dev branch? I fixed this issue in there.

**Reply by Galunid** [reply]
Nope, apparently I pulled latest master. Guess that explains it...
